autoload/rainbow_csv.vim

```diff
@@ -883,12 +892,13 @@ func! rainbow_csv#csv_align()
         let is_first_line = 0
     endfor
     if !has_edit
-        echoerr "File is already aligned"
+        echoerr "Range is already aligned"
```

Comment:
Please keep the original comment since it is a more common use case, and seeing the range might be confusing.

Reply:
done